src/proto/client/mutation.rs

```diff
@@ -0,0 +1,182 @@
+use crate::{
+    proto::single::{MutationAction, MutationType},
+    Client, Error, JobId, MutationFilter, MutationTarget,
+};
+use std::borrow::Borrow;
+
+impl Client {
+    /// Re-enqueue the jobs.
+    ///
+    /// ***Warning!*** The `MUTATE` API is not supposed to be used as part of application logic,
+    /// you will want to use it for administration purposes only.
+    ///
+    /// This method will immediately move the jobs from the targeted set (see [`MutationTarget`])
+    /// to their queues. This will apply to the jobs satisfying the [`filter`](crate::MutationFilter).
```

Comment:
I'm having a hard time following what this operation actually does. For example, with `MutationTarget::Scheduled`, what does it mean for the jobs to "immediately be moved to their queues"? They're already in queues?